src/utils/stringCollector.ts

```diff
@@ -0,0 +1,26 @@
+export default class StringCollector {
```

Comment:
can't find this class uses

do we need it? if yes - do we need this strictly as class? why not a sinle method?

Reply:
String collector removed from the request. Will be added later with tests